client/python/README.md

```diff
@@ -0,0 +1,105 @@
+# `gradio_client`: Use any Gradio app as an API -- in 3 lines of Python
+
+This directory contains the source code for `gradio_client`, a lightweight Python library that makes it very easy to use any Gradio app as an API. This library is CURRENTLY IN ALPHA. APIS MAY CHANGE. WE DO NOT RECOMMEND USING IT EXCEPT IN EXPERIMENTAL PROJECTS. 
+
+As an example, consider the Stable Diffusion Gradio app, which is hosted on Hugging Face Spaces, and which generates images given a text prompt. Using the `gradio_client` library, we can easily use the Gradio as an API to generates images programmatically.
+
+Here's the entire code to do it:
+
+```python
+import gradio_client as grc
+
+client = grc.Client(space="stability-ai/stable-diffusion")
+job = client.predict("a hyperrealistic portrait of a cat wearing cyberpunk armor")
+job.result()
+
+>> URL
```

Comment:
unclear `>> URL` was supposed to be the output from `job.result()`

Reply:
Ok I'll just remove this